scene/animation/animation_tree.cpp

```diff
@@ -77,7 +77,11 @@ void AnimationNode::set_parameter(const StringName &p_name, const Variant &p_val
 
 	const AHashMap<StringName, int>::Iterator it = property_cache.find(p_name);
 	if (it) {
-		process_state->tree->property_map.get_by_index(it->value).value.first = p_value;
+		Pair<Variant, bool> &parameter = process_state->tree->property_map.get_by_index(it->value).value;
```

Comment:
(Very) nitpicky, but for the same code repeated at line 924 you'er using `prop`, and `parameter` here, as the variable name. Could make sense to use the same name for consistency.

Reply:
Unified into `prop`.